### ISSUE TITLE
v055u: UnsupportedOperationException: Failed to resolve attribute at index 5 in NotificationHandlerActivity.handleAccessibilitySettings

### ISSUE THREAD

**Issue by ignoramous** [celzero/rethink-app#2368]
```hs
          Caused by java.lang.UnsupportedOperationException: Failed to resolve attribute at index 5: TypedValue{t=0x2/d=0x7f04045b a=12}
       at android.content.res.TypedArray.getColorStateList(TypedArray.java:598)
       at android.widget.TextView.readTextAppearance(TextView.java:3969)
       at android.widget.TextView.<init>(TextView.java:1064)
       at android.widget.Button.<init>(Button.java:166)
       at android.widget.Button.<init>(Button.java:141)
       at androidx.appcompat.widget.AppCompatButton.<init>(AppCompatButton.java:81)
       at androidx.appcompat.widget.AppCompatButton.<init>(AppCompatButton.java:76)
       at androidx.appcompat.app.AppCompatViewInflater.createButton(AppCompatViewInflater.java:238)
       at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:156)
       at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1654)
       at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1705)
       at android.view.LayoutInflater.tryCreateView(LayoutInflater.java:1071)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1001)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:965)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:1144)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1105)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:1147)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1105)
       at android.view.LayoutInflater.parseInclude(LayoutInflater.java:1284)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:1140)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1105)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:686)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:481)
       at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:751)
       at androidx.appcompat.app.AppCompatDialog.setContentView(AppCompatDialog.java:96)
       at androidx.appcompat.app.AlertController.installContent(AlertController.java:232)
       at androidx.appcompat.app.AlertDialog.onCreate(AlertDialog.java:278)
       at android.app.Dialog.dispatchOnCreate(Dialog.java:423)
       at android.app.Dialog.show(Dialog.java:317)
       at com.celzero.bravedns.ui.NotificationHandlerActivity.handleAccessibilitySettings(NotificationHandlerActivity.kt:192)
       at com.celzero.bravedns.ui.NotificationHandlerActivity.trampoline(NotificationHandlerActivity.kt:114)
       at com.celzero.bravedns.ui.NotificationHandlerActivity.handleNotificationIntent(NotificationHandlerActivity.kt:107)
       at com.celzero.bravedns.ui.NotificationHandlerActivity.onCreate(NotificationHandlerActivity.kt:66)
       at android.app.Activity.performCreate(Activity.java:7893)
       at android.app.Activity.performCreate(Activity.java:7880)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3315)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3489)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2073)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:225)
       at android.app.ActivityThread.main(ActivityThread.java:7563)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:994)
```
```hs
          Caused by android.view.InflateException: Binary XML file line #43 in com.celzero.bravedns:layout/mtrl_alert_dialog_actions: Error inflating class Button

...

                  Caused by android.view.InflateException: Binary XML file line #96 in com.celzero.bravedns:layout/mtrl_alert_dialog: Binary XML file line #43 in com.celzero.bravedns:layout/mtrl_alert_dialog_actions: Error inflating class Button

...

Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.celzero.bravedns/com.celzero.bravedns.ui.NotificationHandlerActivity}: android.view.InflateException: Binary XML file line #96 in com.celzero.bravedns:layout/mtrl_alert_dialog: Binary XML file line #43 in com.celzero.bravedns:layout/mtrl_alert_dialog_actions: Error inflating class Button
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3340)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3489)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2073)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:225)
       at android.app.ActivityThread.main(ActivityThread.java:7563)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:994)
```

**Comment by hussainmohd-a** [comment]
Fixed: 24366416
